index.js

```diff
@@ -0,0 +1,376 @@
+const express = require("express"),
+	dotenv = require("dotenv"),
+	app = express(),
+	path = __dirname + "/views/",
+	bodyParser = require("body-parser"),
+	cookieSession = require("cookie-session"),
+	mysql = require("mysql"),
+	AES = require("mysql-aes"),
+	morgan = require("morgan");
+dotenv.config();
+app.use(morgan("dev"));
+app.use(
+	cookieSession({
+		name: "session",
+		secret: `${process.env.cookie_secret}`,
+		signed: true,
+		maxAge: 1 * 60 * 60 * 1000 // 1 hour
+	})
+);
+app.use(bodyParser.json());
+app.use(
+	bodyParser.urlencoded({
+		extended: false
+	})
+);
+app.set("port", process.env.node_port);
+app.use(express.static(__dirname + "/public"));
+app.set("views", path);
+app.set("view engine", "ejs");
+app.listen(app.get("port"), () => {
+	console.log("App is running on port", app.get("port"));
+});
+const con = mysql.createPool({
+	host: `${process.env.DB_host}`,
+	user: `${process.env.DB_user}`,
+	password: `${process.env.DB_password}`,
+	database: `${process.env.DB_name}`,
+	multipleStatements: true
+});
+// function parseCookies(req) {
+// 	var list = {},
+// 		rc = req.headers.cookie;
+
+// 	rc &&
+// 		rc.split(";").forEach(cookie => {
+// 			var parts = cookie.split("=");
+// 			list[parts.shift().trim()] = decodeURI(parts.join("="));
+// 		});
+
+// 	return list;
+// }
+// var hall_id = parseCookies(req).hall_id;
+app.get("/", (req, res) => {
+	if (req.session.user) {
+		res.redirect("/dashboard");
+	} else res.render("index");
+});
+app.get("/login", (req, res) => {
+	if (req.session.user) {
+		res.redirect("/dashboard");
+	} else res.render("login");
+});
+app.get("/dashboard", (req, res) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Arrow function has a complexity of 5. (complexity)](https://app.codacy.com/app/AravindVNair99/Hall-Management-System/pullRequest?prid=4592231)